libs/sdk-ui-charts/tsconfig.json

```diff
@@ -21,7 +21,7 @@
             "@gooddata/sdk-model": ["sdk-model/src"],
             "@gooddata/sdk-backend-spi": ["sdk-backend-spi/src"],
             "@gooddata/sdk-backend-mockingbird": ["sdk-backend-mockingbird/src"],
-            "@gooddata/util": ["util/src"]
+            "@gooddata/util": ["util/src"],
```

Comment:
comma probably should not be there (last line in the json block)

Reply:
fixed